src/booleans.js

```diff
@@ -1,62 +1,82 @@
 function negate(a) {
-  // your code here
-};
+  return !a;
+}
 
 function both(a, b) {
-  // your code here
-};
+  return a && b;
+}
 
 function either(a, b) {
-  // your code here
-};
+  return a || b;
+}
 
 function none(a, b) {
-  // your code here
-};
+  return !(a || b);
+}
 
 function one(a, b) {
-  // your code here
-};
+  if (a && b) {
+    return false;
+  }
+  if (a || b) {
+    return true;
+  }
+  return false;
```

Comment:
This could simplified a bit. The requirement is to check is one or the other (but not both) values are true, which could written as:
```suggestion
  return (a && !b) || (b && !a);
```